draft-vvv-webtransport-quic.md

```diff
@@ -233,6 +233,42 @@ datagram ID SHALL be absent.
 
 The datagrams sent using QuicTransport MUST be subject to congestion control.
 
+# QuicTransport URI Scheme {#uri}
+
+NOTE: the URI scheme defintion in this section is provisional and subject to
+change, especially the name of the scheme.
+
+QuicTransport uses `quic-transport` URI scheme for identifying QuicTransport
+servers.
+
+The syntax definition below uses Augmented Backus-Naur Form (ABNF) {{!RFC5234}}.
+The definitions of `host`, `port`, `path-abempty`, `query` and `fragment` are
+adopted from {{!RFC3986}}.  The syntax of a QuicTransport URI SHALL be:
+
+~~~~~~~~~~~~~~~
+quic-transport-URI = "quic-transport:" "//"
+                             host [ ":" port ]
+                             [ "?" query ]
+                             [ "#" fragment ]
+~~~~~~~~~~~~~~~
+
+This document does not define any semantics to `path-abemtpy`, `query` or
+`fragment` portions of the URI, nor does it delegate those to the host owners.
+Any QuicTransport implementation MUST ignore those until a subsequent
+specification assigns semantics to those.
+
+The `host` component MUST NOT be empty.  If the `port` component is missing, the
+port SHALL be assumed to be 0.
+
+In order to connect to a QuicTransport server identified by a given URI, the
+user agent SHALL establish a QUIC connection to the specified `host` and `port`
+as described in {{connection}}.  It MUST immediately signal an error to the user
```

Comment:
I don't think we've defined what a user is. Perhaps replace this with `If the port value is 0, the user agent MUST fail the connection attempt immediately.`

Reply:
I think I've defined `client` to mean what I meant here, so I just replaced it with that.